docs/installation-and-operations/installation/docker/README.md

```diff
@@ -78,7 +78,11 @@ The fastest way to get an OpenProject instance up and running is to run the
 following command:
 
 ```bash
-docker run -it -p 8080:80 -e OPENPROJECT_SECRET_KEY_BASE=secret -e OPENPROJECT_HOST__NAME=localhost:8080 openproject/community:12
+docker run -it -p 8080:80 \
+  -e OPENPROJECT_SECRET_KEY_BASE=secret \
+  -e OPENPROJECT_HOST__NAME=localhost:8080 \ # Set the host name to what you will externally use it with
+  -e OPENPROJECT_HTTPS=false \ # Disable HTTPS due to accessing container directly
+  openproject/community:12
 ```
 
```

Comment:
Having `\` to split lines wont work if followed by `#`.

I can't propose a good suggestion because of the triple backticks in the edited code, but I would suggest adding another paragraph after the command explaining why `OPENPROJECT_HOST__NAME=localhost:8080` and `OPENPROJECT_HTTPS=false` are set.